index.html

```diff
@@ -5,44 +5,129 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.1/dist/css/bootstrap.min.css" rel="stylesheet">
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.1/dist/js/bootstrap.bundle.min.js"></script>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.0/css/all.min.css" integrity="sha512-xh6O/CkQoPOWDdYTDqeRdPCVd1SpvCA9XXcUnZS2FmJNp1coAFzvtCN9BmamE+4aHK8yyUHUSCcJHgXloTyT2A==" crossorigin="anonymous" referrerpolicy="no-referrer" />
     <link rel="stylesheet" href="css/style.css">
     <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Lato">
 </head>
 <body>
    <header>
+       <div class="upper_navbar">
+           <ul>
+            <li><a href="#"><img src="images/linkedinlogo.png" alt="Linked in nav item"></a></li>
+            <li><a href="#"><img src="images/twitterlogo.png" alt="Linked in nav item"></a></li>
+            <li><a href="#">English</a></li>
+            <li><a href="#">My Page</a></li>
+            <li><a href="#">Logout</a></li>          
+         </ul>
+       </div>
        <nav>
-           <button type="button"><img src="images/hamburger.png" alt="Hamburger button"></button>
-       </nav>
+           <button type="button" id="hamburger-btn"><img src="images/hamburger.png" alt="Hamburger button"></button>
+           <div class="desktop-navbar">
+                <a class="logo_link" href="#"><img src="images/Screenshot from 2022-02-16 15-13-24.png" alt="Homifyers logo link"></a>
+                <ul>
+                    <li><a href="#">Home</a></li>
+                    <li><a href="about.html">About</a></li>
+                    <li><a href="#categories">Categories</a></li>
+                    <li><a href="#tstmons">Top Customers</a></li>
+                    <li><a href="#brands">Brands</a></li>
+                    <li><a href="#wrhs">Visit</a></li>
+                </ul>
+            </div>
+        </nav>
    </header>
    <main>
-       <sectiom class="hero">
+       <section class="hero" id="hero">
             <h3>Hello! Online Available Furniture</h3>
             <h1>Beautifully Designed Furniture</h1>
             <p class="msg">A website where all kind of furniture is available. Beatifully designed and best furniture is available and it's delivery is also available. 100%% best quality furniture is available at your door step.
                 You can choose and buy your favourite furniture from our website and we can deliver it to you.
             </p>
-            <h3 class=""></h3>
-            <p>@Furnishers Islamabad, Pakistan</p>
-       </sectiom>
-       <section class="works">
+            <h2 class="current-time"></h2>
+            <p class="tit">@Furnishers Islamabad, Pakistan</p>
+       </section>
```

Comment:
- [ ] The top section background image is not aligned properly, kindly consider adding the background to the top section only so it can look like the provided design

|Your design|Provided design|
|-|-|
|![Screenshot_20221028_224105](https://user-images.githubusercontent.com/25227022/198737526-e274a289-691a-4350-b53e-02bc77260248.png)|![Screenshot_20221028_224221](https://user-images.githubusercontent.com/25227022/198737606-517b7ece-d963-46d1-865f-3fcbc4e09490.png)|